### PR TITLE
Added empty state to RuleTable

### PR DESCRIPTION
**Pull request by bond95**
image

@karelhala @doruskova please, take a look

### CODE REVIEW

File: packages/rule-components/src/RuleTable/RuleTable.js [hunk 1]
```diff
@@ -177,9 +179,31 @@ class RuleTable extends Component {
...
                             }]
                         })) :
-                        flatten(createRows(data, columns, expanded, detail)).filter(Boolean)
+                        data.length > 0 ?
+                            flatten(createRows(data, columns, expanded, detail)).filter(Boolean) :
+                            [{
+                                heightAuto: true,
+                                cells: [
+                                    {
+                                        props: { colSpan: 8 },
```
Comment (karelhala) [inline packages/rule-components/src/RuleTable/RuleTable.js:188]: This should be length of columns, so anybody can suppply whatever amount of columns they want.

File: packages/rule-components/src/RuleTable/RuleTable.js [hunk 2]
```diff
@@ -177,9 +179,31 @@ class RuleTable extends Component {
...
+                                        title: (
+                                            <Bullseye>
+                                                <EmptyState variant={EmptyStateVariant.small}>
+                                                    <EmptyStateIcon icon={SearchIcon} />
+                                                    <Title headingLevel="h2" size="lg">
+                                                        No results found
```
Comment (karelhala) [inline packages/rule-components/src/RuleTable/RuleTable.js:194]: I would have make these props with default value to these strings so anybody can supply their own strings.

File: packages/rule-components/src/RuleTable/RuleTable.js [hunk 3]
```diff
@@ -177,9 +179,31 @@ class RuleTable extends Component {
...
+                            flatten(createRows(data, columns, expanded, detail)).filter(Boolean) :
+                            [{
+                                heightAuto: true,
+                                cells: [
+                                    {
+                                        props: { colSpan: 8 },
+                                        title: (
+                                            <Bullseye>
+                                                <EmptyState variant={EmptyStateVariant.small}>
+                                                    <EmptyStateIcon icon={SearchIcon} />
```
Comment (karelhala) [inline packages/rule-components/src/RuleTable/RuleTable.js:192]: Can you add `emptyIcon` property so anybody using this component can change it. You can assign `SearchIcon` as default prop.

**Comment by bond95** [comment]
@karelhala updated

**Comment by doruskova** [comment]
@bond95 looks good!

**Pull request merged**